packages/hardhat-foundry/README.md

```diff
@@ -0,0 +1,7 @@
+[![npm](https://img.shields.io/npm/v/@nomicfoundation/hardhat-foundry.svg)](https://www.npmjs.com/package/@nomicfoundation/hardhat-foundry) [![hardhat](https://hardhat.org/buidler-plugin-badge.svg?1)](https://hardhat.org)
+
+# hardhat-foundry
+
+This plugin makes it easier to use Hardhat and Foundry in the same project.
```

Comment:
I don't think this is enough. It should explain what is the intended output/behavior of the plugin.

Note that this is what users will see if they land to the plugin page in hardhat.org or npm

Reply:
I expanded the README a bit, let me know what you think.